docs/reference/planners/Web-Planner/main.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: default
+title: web-planner
+parent: Planners
+permalink: /ref/planners/web-planner
+nav_order: 100
+---
+# Web-Planner
+
+Page Contributors: {% git_author %}
+
+{% planner_tag Optimising Satisfycing %}
+
+Year Published: 2017
+
+Paper: [WEB PLANNER: A Tool to Develop Classical Planning Domains and Visualize Heuristic State-Space Search](http://icaps17.icaps-conference.org/workshops/UISP/uisp17proceedings.pdf#page=36) [Magnaguagno, M. Pereira, R. M�re, M. and Meneguzzi, F.]
+
+Similar to [Planning.Domains](#planning.domains), it is a planner in the cloud with editor, verification, validation and visualization capabilities.
+The planner is available at https://web-planner.herokuapp.com/
+For more information see:
+- [DOVETAIL - An Abstraction for Classical Planning Using a Visual Metaphor](https://www.aaai.org/ocs/index.php/FLAIRS/FLAIRS16/paper/viewFile/12966/12544) - FLAIRS 2016
```

Comment:
A relevant paragraph should be included rather than a link to other material

Reply:
I added more content to describe the system itself, not sure if I should add a paragraph for each related publication.